rcsb/utils/insilico3d/ModelArchiveModelProvider.py

```diff
@@ -144,8 +158,7 @@ def __reload(self, **kwargs):
                 for dataSet, pathD in modelArchiveRequestedDatasetD.items():
                     try:
                         sD = {}
-                        dataSetNumModels = pathD.get("numModelsTotal")
-                        numModelsToDownload = pathD.get("numModels", dataSetNumModels)  # Used for testing purposes, defaults to total number of models
+                        numModelsToDownload = pathD.get("numModels", None)  # Used for testing purposes, defaults to total number of models
                         bulkFileName = pathD.get("bulkFileName", None)
```

Comment:
Am I right in understanding that neither full download or test download use "bulkFileName".

Reply:
Currently, no. I've left the functionality in there to perform the bulk file download once bulk files are available for everything, but for now I'm defaulting things to perform the individual-file download instead.